### PR TITLE
Fix conpoment props`s value show not correct in pages when the value is  not in English

### LINKED ISSUE #282: Stream did not contain valid UTF-8
**Describe the bug**
When characters such as Chinese characters and Hangul are initially rendered, they are converted into invalid characters and entered in index.js. Therefore, a stream did not contain valid UTF-8 error occurs.

For example:
```
def index():
    return pc.container(
        pc.text("한글")
    )

app = pc.App()
app.add_page(index)
app.compile()
```
this code makes an error

```
class State(pc.State):
    text = "한글"

def index():
    return pc.container(
        pc.text(State.text)
    )

app = pc.App(state=State)
app.add_page(index)
app.compile()
```

but this works.
This obviously looks like a bug.

**Screenshots**
invalid

index.js when the first code was compiled

** Specifics (please complete the following information):**
 - Python Version: 3.10.xx
 - Pynecone Version: 0.1.12
 - OS: Windows

### PR DESCRIPTION
**Pull request by DeathShort**
### All Submissions:

- [ ] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [ ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

### Changes To Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.
    Fix issue #282 .This issue is caused by pynecone.components.tags.tag ,when format the props, the json.dumps function use ascii encode as default. 

    b. Describe your changes.
    The solution is add 'ensure_ascii=False' in every case of json.dumps uses.
    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: pynecone/components/tags/tag.py [hunk 1]
```diff
@@ -62,20 +63,21 @@ def format_prop(
             if not prop.is_local or prop.is_string:
                 return str(prop)
             if issubclass(prop.type_, str):
-                return json.dumps(prop.full_name)
+                return json.dumps(prop.full_name, ensure_ascii=False)
```
Comment (picklelo) [inline pynecone/components/tags/tag.py:66]: Might be worth it to add a helper function in `utils.py` so we never forget to add this.

**Review by picklelo — COMMENTED** [review]
Looks good! You need to run the linter. See the bottom here: https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md

File: pynecone/components/tags/tag.py (hunk 1 above)
Comment (DeathShort) [reply pynecone/components/tags/tag.py:65]: Ok，I ran the linter as the doc said and  commit again , please review the code.
And by the way , your guys did a fantastic job, I love this project and wish it goes well!

**Review by picklelo — APPROVED** [review]
looks good!

**Pull request merged**